### DISCUSSION TITLE
[Projects Beta] Group by repository

### DISCUSSION THREAD

**Discussion by pavlenex** [community/community#7418] [Projects and Issues] [answered]
For some reasons, it's not possible to group by `repository`. As a team which has plenty of repos, this would be a great functionality for us.
![Screenshot 2021-11-07 at 11 25 57](https://user-images.githubusercontent.com/36959754/140641223-f1c96170-8a8b-4ee6-b6cc-ce44f9733f31.png)

**Comment by lysut** [comment]
Looks like this might be handled in [#283](https://github.com/github/roadmap/issues/283)

**Comment by azenMatt** [comment, marked as answer]
Hi @pavlenex @lysut  - this is now live :) 

you can check out the other latest updates here - https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/
